docs/lcb/notes/feature-java.md

```diff
@@ -0,0 +1,15 @@
+# LiveCode Builder Standard Library
+
+## Java utilities
+
+There is now a utility library for manipulating java objects. It contains
+a type JObject which wraps a java object, some type conversion operations:
```

Comment:
"java" -> "Java"
I suggest putting `JObject` (and other code symbols) in `code sample markup`